src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -44,7 +43,7 @@ import StartTimeSelection from './components/StartTimeSelection';
 import TravellerSelection from './components/TravellerSelection';
 import FullScreenFooter from '@atb/components/screen-footer/full-footer';
 import {getPurchaseFlow} from '../utils';
-import {getDeviceNameForCurrentToken} from '@atb/mobile-token/utils';
+import {getOtherDeviceIsInspectableWarning} from '../../Ticket/utils';
```

Comment:
Same as https://github.com/AtB-AS/mittatb-app/pull/2892/files#r954826095

```suggestion
import {getOtherDeviceIsInspectableWarning} from '@atb/screens/Ticketing/Ticket/utils';
```